test/lib/optionMap.ts

```diff
@@ -0,0 +1,23 @@
+import { RendererOptions } from "quicktype-core";
+import { ComparisonArgs } from "../utils";
+import { GoLanguage, Language } from "../languages";
+
+type TestOptions = {
+    cliOptions: RendererOptions;
+    language: Language;
+    comparisonArgs?: Partial<ComparisonArgs>;
+};
+
+const optionMap: Record<string, TestOptions> = {
+    "test/inputs/json/misc/227e2.in.json": {
```

Comment:
This means, for `227e2.in.json`, the test will be run with `omit-empty` set to true, strict comparison of input and expected output, and will only run for Golang language.